openmdao_book/other_useful_docs/auto_ivc_api_translation.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<p>Shouldn't it be with a comma?</p><p> </p><blockquote>release, it is</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Fixed</p><p> </p>